vignettes/minimal-rstan-package.Rmd

```diff
@@ -235,13 +235,14 @@ information about the package.
 
 With __roxygen__ documentation enabled, we need to generate the documentation
 for `lm_stan` and update the `NAMESPACE` so the function is exported, i.e.,
-available to users when the package is installed.  This can be done with the
-function `roxygen2::roxygenize()`, which needs to be called twice initially.
+available to users when the package is installed. This can be done with the
+function `devtools::document()`.
```

Comment:
I think we decided to change this separately from this PR. Looks like the code change was already reverted but the text still mentions `document()`.